### ISSUE TITLE
[FEATURE]: Add a folder to separate TypeScipt components and JavaScript components

### ISSUE THREAD

**Issue by AbhijithGanesh** [ComputerSocietyVITC/Annual-Yearbook#23]
### Description

# Addition of separate components

The current directory structures the component in the following way:
```
├── components
│   ├── contributors.tsx
│   ├── header.jsx
│   ├── layout.jsx
│   └── main.jsx
├── images
│   └── icon.png
├── pages
│   ├── 404.js
│   ├── index.js
│   └── {year}.jsx
├── styles
│   └── global.css
└── types
    └── ContributorType.ts
``` 

We suggest that the components be separated into files dependent on their language

```
.
├── components
│   ├── javascript
│   │   ├── header.jsx
│   │   ├── layout.jsx
│   │   └── main.jsx
│   └── typescript
│       └── contributors.tsx
├── images
│   └── icon.png
├── pages
│   ├── 404.js
│   ├── index.js
│   └── {year}.jsx
├── styles
│   └── global.css
└── types
    └── ContributorType.ts


```

## TL;DR
- [ ] Make a folder for storing TypeScript files
- [ ] Make a folder for storing JavaScript for files

### Expected Behaviour

Should store components separately!

**Comment by prerit2001** [comment]
Please Assign me this.

**Comment by priyanshusingh305** [comment]
please assign me

**Comment by AbhijithGanesh** [comment]
Hello @prerit2001 and @priyanshusingh305 ! Welcome to the project ❣️. @priyanshusingh305 can you take a look at the other issues like #21

**Comment by cool-pants** [comment]
@prerit2001 have you started working on this?

**Comment by cool-pants** [comment]
If you haven't, then could you add the changes as PR to [this](https://github.com/cool-pants/Annual-Yearbook/tree/mainak) repo branch, since the typescript integration PR hasn't been merged in prod as of yet.

**Comment by AbhijithGanesh** [comment]
@cool-pants They can rebase their code-base. We will add an option to automatically rebase.

**Comment by AbhijithGanesh** [comment]
@prerit2001 Please continue your work and send in your merge-request to the master itself! Thanks

**Comment by prerit2001** [comment]
Can you please merge the changes and label this with hacktober-accepted.

**Comment by AbhijithGanesh** [comment]
Thank you for your contribution @prerit2001
